spotbugs/src/main/java/edu/umd/cs/findbugs/ba/NullnessAnnotation.java

```diff
@@ -54,14 +55,23 @@ boolean match(@DottedClassName String className) {
         boolean match(@DottedClassName String className) {
             // Unfortunately there are mixed case Nonnull and NonNull annotations (JSR305, FB and JDT)
             return "org.jetbrains.annotations.NotNull".equals(className)
+                    // TODO: Need to be support transitive as per https://github.com/jspecify/jspecify/blob/main/src/main/java/org/jspecify/annotations/NullUnmarked.java#L29
```

Comment:
Are you sure, that this link is correct? Shouldn't it refer to `NullMarked` and not to `NullUnmarked`?